pkg/operator/baremetal_test.go

```diff
@@ -27,7 +27,7 @@ func newOperatorWithBaremetalConfig() *OperatorConfig {
 			"quay.io/openshift/origin-ironic:v4.2.0",
 			"quay.io/openshift/origin-ironic-inspector:v4.2.0",
 			"quay.io/openshift/origin-ironic-ipa-downloader:v4.2.0",
-			"quay.io/openshift/origin-ironic-rhcos-downloader:v4.2.0",
+			"quay.io/openshift/origin-ironic-machine-os-downloader:v4.2.0",
```

Comment:
I don't understand the `:v4...` suffixes, but for the rest of this PR you have `v4.3.0`, while here you have `v4.2.0`.  Maybe these should all be replaced with dummy placeholders (e.g. see openshift/machine-config-operator#750)?

Reply:
I agree but sounds unrelated to this PR, there's a lot of mismatches in the suffixes, it seems whenever people add them they do whatever was current, there are examples of v4.0, v4.2, etc in here:

```
pkg/operator/operator_test.go:			"docker.io/openshift/origin-aws-machine-controllers:v4.0.0",
pkg/operator/operator_test.go:			"docker.io/openshift/origin-machine-api-operator:v4.0.0",
pkg/operator/operator_test.go:			"docker.io/openshift/origin-machine-api-operator:v4.0.0",
pkg/operator/config.go:	// TODO(alberto): move to "quay.io/openshift/origin-kubemark-machine-controllers:v4.0.0" once available
pkg/operator/config_test.go:	expectedAWSImage                = "docker.io/openshift/origin-aws-machine-controllers:v4.0.0"
pkg/operator/config_test.go:	expectedLibvirtImage            = "docker.io/openshift/origin-libvirt-machine-controllers:v4.0.0"
pkg/operator/config_test.go:	expectedOpenstackImage          = "docker.io/openshift/origin-openstack-machine-controllers:v4.0.0"
pkg/operator/config_test.go:	expectedMachineAPIOperatorImage = "docker.io/openshift/origin-machine-api-operator:v4.0.0"
pkg/operator/config_test.go:	expectedBareMetalImage          = "quay.io/openshift/origin-baremetal-machine-controllers:v4.0.0"
pkg/operator/config_test.go:	expectedAzureImage              = "quay.io/openshift/origin-azure-machine-controllers:v4.0.0"
pkg/operator/config_test.go:	expectedGCPImage                = "quay.io/openshift/origin-gcp-machine-controllers:v4.0.0"
pkg/operator/config_test.go:	expectedBaremetalOperator       = "quay.io/openshift/origin-baremetal-operator:v4.2.0"
pkg/operator/config_test.go:	expectedIronic                  = "quay.io/openshift/origin-ironic-image:v4.2.0"
pkg/operator/config_test.go:	expectedIronicInspector         = "quay.io/openshift/origin-ironic-inspector-image:v4.2.0"
pkg/operator/config_test.go:	expectedIronicIpaDownloader     = "quay.io/openshift/origin-ironic-ipa-downloader:v4.2.0"
pkg/operator/config_test.go:	expectedIronicRhcosDownloader   = "quay.io/openshift/origin-ironic-rhcos-downloader:v4.2.0"
pkg/operator/config_test.go:	expectedIronicStaticIpManager   = "quay.io/openshift/origin-ironic-static-ip-manager:v4.2.0"
pkg/operator/fixtures/images.json:  "clusterAPIControllerAWS": "docker.io/openshift/origin-aws-machine-controllers:v4.0.0",
pkg/operator/fixtures/images.json:  "clusterAPIControllerOpenStack": "docker.io/openshift/origin-openstack-machine-controllers:v4.0.0",
pkg/operator/fixtures/images.json:  "clusterAPIControllerLibvirt": "docker.io/openshift/origin-libvirt-machine-controllers:v4.0.0",
pkg/operator/fixtures/images.json:  "machineAPIOperator": "docker.io/openshift/origin-machine-api-operator:v4.0.0",
pkg/operator/fixtures/images.json:  "clusterAPIControllerBareMetal": "quay.io/openshift/origin-baremetal-machine-controllers:v4.0.0",
pkg/operator/fixtures/images.json:  "clusterAPIControllerAzure": "quay.io/openshift/origin-azure-machine-controllers:v4.0.0",
pkg/operator/fixtures/images.json:  "clusterAPIControllerGCP": "quay.io/openshift/origin-gcp-machine-controllers:v4.0.0",
pkg/operator/fixtures/images.json:  "baremetalOperator": "quay.io/openshift/origin-baremetal-operator:v4.2.0",
pkg/operator/fixtures/images.json:  "baremetalIronic": "quay.io/openshift/origin-ironic-image:v4.2.0",
pkg/operator/fixtures/images.json:  "baremetalIronicInspector": "quay.io/openshift/origin-ironic-inspector-image:v4.2.0",
pkg/operator/fixtures/images.json:  "baremetalIpaDownloader": "quay.io/openshift/origin-ironic-ipa-downloader:v4.2.0",
pkg/operator/fixtures/images.json:  "baremetalRhcosDownloader": "quay.io/openshift/origin-ironic-rhcos-downloader:v4.2.0",
pkg/operator/fixtures/images.json:  "baremetalStaticIpManager": "quay.io/openshift/origin-ironic-static-ip-manager:v4.2.0"
```

Reply:
The :v4.* are image tags in quay.io. Yes, it appears that the these image versions almost represents the last release these were updated.

Reply:
@wking I created a ticket to consolidate on "more" dummy placeholders